ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -443,10 +446,13 @@ void combine_nets(nnet_t* output_net, nnet_t* input_net, netlist_t* netlist) {
  *-------------------------------------------------------------------------------------------*/
 void join_nets(nnet_t* join_to_net, nnet_t* other_net) {
     if (join_to_net == other_net) {
-        if ((join_to_net->driver_pin) && (join_to_net->driver_pin->node != NULL) && join_to_net->driver_pin->node->related_ast_node != NULL)
-            error_message(NETLIST, join_to_net->driver_pin->node->related_ast_node->line_number, join_to_net->driver_pin->node->related_ast_node->file_number, "%s", "This is a combinational loop\n");
+#warning "Oops this be broke come fix"
+        oassert(join_to_net->num_driver_pins <= 1);
+        if ((join_to_net->num_driver_pins) && (join_to_net->driver_pins[0]->node != NULL) && join_to_net->driver_pins[0]->node->related_ast_node != NULL)
```

Comment:
could you for loop to check?

Reply:
Yeah no idea why I didnt in the first place. Also those error messages look pretty like they could do with a cleanup? Or does `This is a combinational loop with more info` have some special meaning

Reply:
Also should I be looping over the fanout pins while I am at it?